app/src/main/java/com/ichi2/apisample/ui/AudioCaptureService.java

```diff
@@ -202,6 +210,7 @@ public void onClick(View view) {
                         .setUsage(AudioAttributes.USAGE_MEDIA)
                         .build()
         );
+        toneGenerator = new ToneGenerator(AudioManager.STREAM_SYSTEM, 25);
```

Comment:
Please use named constants for such an unobvious numeric values (here and below)

Reply:
Fixed